acj/__init__.py

```diff
@@ -68,6 +69,13 @@ def unauthorized():
     # Flask-Bouncer initialization
     bouncer.init_app(app)
 
+    celery = Celery(
+        app.name,
+        backend=app.config['CELERY_RESULT_BACKEND'],
+        broker=app.config['CELERY_BROKER_URL']
+    )
+    celery.conf.update(app.config)
```

Comment:
is this update override the CELERY_RESULT_BACKEND and CELERY_BROKER_URL? if yes, do you need to set them in line 74 and 75?